packages/shared/src/utils/stateless/calendar/calendar-event/calendar-event.impl.ts

```diff
@@ -72,6 +73,7 @@ export default class CalendarEventImpl implements CalendarEventInternal {
 
   get _isSingleHybridDayTimed(): boolean {
     if (!this._config.isHybridDay) return false
+    if (this.start === this.end) return false
```

Comment:
why do we need this line?

I believe this getter can never be true, if two timed events are identical